llvm/include/llvm/CodeGen/GlobalISel/InstructionSelector.h

```diff
@@ -36,6 +38,8 @@ class InstructionSelector : public GIMatchTableExecutor {
   const TargetPassConfig *TPC = nullptr;
 
   MachineOptimizationRemarkEmitter *MORE = nullptr;
+
+  GISelObserverWrapper *AllObservers = nullptr;
```

Comment:
Add `/// Note: InstructionSelect does not track changed instructions. changingInstr() and changedInstr() will never be called on these observers.`?